weblate/settings_example.py

```diff
@@ -29,13 +29,13 @@
 SITE_DOMAIN = ""
 
 # Whether site uses https
-ENABLE_HTTPS = False
+ENABLE_HTTPS = True
 
 #
 # Django settings for Weblate project.
 #
 
-DEBUG = True
+DEBUG = False
```

Comment:
The debug is intentionally enabled for development purposes. The same applies to HTTPS, which is not supported by Django built-in server.

Reply:
If HTTPS isn't supported, why does it exist in the config?

Reply:
It is not supported by Django built-in server, which is suitable for development only.